src/main/java/edu/harvard/iq/dataverse/authorization/users/User.java

```diff
@@ -14,4 +14,6 @@
 
     public boolean isSuperuser();
 
+    public boolean isDisabled();
```

Comment:
fyi, it is now possible to add a default implementation of the method and so not have to override in subclasses if they use the default.

Reply:
Gotcha. Do you want me to try to make this change for isDisabled? (I'd rather not mess around with isAuthenticated and isSuperuser right now).

Reply:
Yeah, let's do it. It's not important enough to hold anything up, but should a good tool to use for these kinds of things in the future.

Reply:
Done in 23b8bf638.